packages/mux-player/src/themes/2023/2023.html

```diff
@@ -732,6 +734,48 @@
     <media-time-range part="bottom time range" disabled="{{disabled}}" aria-disabled="{{disabled}}"></media-time-range>
   </template>
 
+  <template partial="AudioTrackSelect">
+    <media-audio-track-selectmenu
+      part="bottom audio-track selectmenu"
+      disabled="{{disabled}}"
+      aria-disabled="{{disabled}}"
+    >
+      <media-audio-track-button slot="button" part="bottom audio-track button">
+        <svg aria-hidden="true" slot="icon" viewBox="0 0 24 24">
```

Comment:
All of our current theme icons are 18 x 14 so this has a ratio that doesn't match. I don't think it's a problem for merging it in now but I'll make a note to get an icon from Jason, I think we need a new icon for rendition selector anyway as we're currently using the very general triple dot one.

Reply:
cool agree with everything. Also the icon may change based on out of band convo, so probably don't need to belabor it too much on this PR as long as it's resolved before switching 2023 to default.